BrainPortal/app/models/userfile.rb

```diff
@@ -692,7 +692,8 @@ def self.find_or_create_as_scratch(attributes = {}, &block)
 
   class Viewer #:nodoc:
 
-    attr_reader :userfile_class, :name, :partial
+    attr_reader   :userfile_class, :name, :partial, :conditions
```

Comment:
Are accessed to @conditions ever performed by method calls? If not remove :conditions here